### DISCUSSION TITLE
Playback speed

### DISCUSSION THREAD

**Discussion by rmoncks** [URUWorks/TeroSubtitler#317] [Ideas]
Hi guys,

First of all, thanks so much for this great app! Since I'm using it to review subtitles, one thing that would be amazing is to change playback speed (to watch the video in 1.5x or 2x). I'm not sure if this is already available or if it has been discussed before, but unfortunately I haven't found it. If it's not available, Is this possible to implement?

Thanks!
Rodrigo

**Comment by URUWorks** [comment]
This function is implemented.

You must activate the "Other" toolbar:
![01](https://github.com/URUWorks/TeroSubtitler/assets/19515662/ebc82c8d-9673-4679-a30d-fa5503566c6c)

The playrate button appears to activate or deactivate the playback speed, in the submenu of said button you can choose the percentage of the speed:
![02](https://github.com/URUWorks/TeroSubtitler/assets/19515662/b7985599-8d3e-43f3-a54c-9d17f9949d12)
